server/app/views/NorthStarBaseView.java

```diff
@@ -65,6 +65,7 @@ protected ThymeleafModule.PlayThymeleafContext createThymeleafContext(
     ThymeleafModule.PlayThymeleafContext context = playThymeleafContextFactory.create(request);
     context.setVariable("civiformImageTag", settingsManifest.getCiviformImageTag().get());
     context.setVariable("addNoIndexMetaTag", settingsManifest.getStagingAddNoindexMetaTag());
+    context.setVariable("favicon", settingsManifest.getFaviconUrl().get());
```

Comment:
```suggestion
    context.setVariable("favicon", settingsManifest.getFaviconUrl().orElse("");
```

(blocking) If `getFaviconUrl()` is an empty Optional this will raise an error. Using orElse("") will default it to an empty string instead so that your thymeleaf `th:if` will then not render the link if it's empty.